libr/bin/format/mach0/coresymbolication.c

```diff
@@ -0,0 +1,306 @@
+#include <r_types.h>
+#include <r_util.h>
+#include <r_hash.h>
+#include "coresymbolication.h"
+
+#define R_CS_EL_OFF_SEGS 0x58
+#define R_CS_EL_SIZE_SEG 0x20
+#define R_CS_EL_SIZE_SECT_64 0x18
+#define R_CS_EL_SIZE_SECT_32 0x10
+#define R_CS_EL_SIZE_SYM 0x18
+#define R_CS_EL_SIZE_LSYM 0x24
+#define R_CS_EL_SIZE_LINFO 0x14
+
+static RCoreSymCacheElementHdr *r_coresym_cache_element_header_new(RBuffer *buf, size_t off, int bits) {
+	RCoreSymCacheElementHdr *hdr = R_NEW0 (RCoreSymCacheElementHdr);
+	if (!hdr) {
+		return NULL;
+	}
+	if (r_buf_fread_at (buf, off, (ut8 *)hdr, "13i16c5i", 1) != sizeof (RCoreSymCacheElementHdr)) {
+		free (hdr);
+		return NULL;
+	}
+	return hdr;
+}
+
+void r_coresym_cache_element_free(RCoreSymCacheElement *element) {
```

Comment:
static

Reply:
that's exposed in `coresymbolication.h` and used by `bin_symbols.c`